core/src/mill/util/ClassLoader.scala

```diff
@@ -5,22 +5,33 @@ import java.net.{URL, URLClassLoader}
 import io.github.retronym.java9rtexport.Export
 
 object ClassLoader {
-  def create(urls: Seq[URL],
-             parent: java.lang.ClassLoader)
-            (implicit ctx: Ctx.Home): URLClassLoader = {
-    val cl = new URLClassLoader(urls.toArray, parent)
-    if (!ammonite.util.Util.java9OrAbove) return cl
-    try {
-      cl.loadClass("javax.script.ScriptEngineManager")
-      cl
-    } catch {
-      case _: ClassNotFoundException =>
-        val path = ctx.home
-        val rtFile = new java.io.File(path.toIO, s"rt-${System.getProperty("java.version")}.jar")
-        if (!rtFile.exists) {
-          java.nio.file.Files.copy(Export.export().toPath, rtFile.toPath)
-        }
-        new URLClassLoader((urls ++ Some(rtFile.toURI.toURL)).toArray, parent)
+  def create(urls: Seq[URL], parent: java.lang.ClassLoader)(
+      implicit ctx: Ctx.Home): URLClassLoader = {
+    if (ammonite.util.Util.java9OrAbove) {
+      val platformParent =
```

Comment:
In java 9 we should use `ClassLoader.getPlatformClassLoader` instead of null when creating new classloader. https://stackoverflow.com/a/46494286